AIPscan/Aggregator/tests/test_mets.py

```diff
@@ -13,19 +13,30 @@
 
 
 @pytest.mark.parametrize(
-    "fixture_path, transfer_name",
+    "fixture_path, transfer_name, mets_error",
     [
-        (os.path.join("features_mets", "features-mets.xml"), "myTransfer"),
-        (os.path.join("iso_mets", "iso_mets.xml"), "iso"),
+        (os.path.join("features_mets", "features-mets.xml"), "myTransfer", False),
+        (os.path.join("iso_mets", "iso_mets.xml"), "iso", False),
+        (os.path.join("original_name_mets", "dataverse_example.xml"), "", True),
```

Comment:
Am I understanding correctly that this test raises a `METSError` because Archivematica has written two `dmdSec`s with an ID of `"dmdSec_1"` and mets-reader-writer isn't able to read two `dmdSec`s with the same ID? Forgive me if this naive, but with the naked eye it looks like like the `<premis:originalName>` value in the second `<mets:dmdSec ID="dmdSec_1">` (starting on line 39) should be what we're looking for.

Reply:
That is correct. I will add notes to the test. Related issue: https://github.com/archivematica/Issues/issues/1310